app/services/map_marker_json.rb

```diff
@@ -1,9 +1,18 @@
 module MapMarkerJson
-  def self.build(organisations)
-    Gmaps4rails.build_markers(Queries::Organisations.add_recently_updated_and_has_owner(organisations)) do |org, marker|
-      yield org, marker
-    end.select do |marker|
-      marker[:lat].present? && marker[:lng].present?
-    end.to_json
+  def self.build(orgs, include_extra_organisation_data = true)
+    orgs = organisations_with_extra_data(orgs) if include_extra_organisation_data
+    Gmaps4rails.build_markers(orgs) { |*args| yield *args }
+      .select { |marker| lat_lng_present?(marker) }.to_json
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)